src/bytes.rs

```diff
@@ -42,7 +42,8 @@ glib_wrapper! {
 
 impl Bytes {
     /// Copies `data` into a new shared slice.
-    fn new(data: &[u8]) -> Bytes {
+    fn new<T: AsRef<[u8]>>(data: T) -> Bytes {
```

Comment:
This is also something we should teach GIR about

Reply:
Not sure that we need transfer ownership in this case

Reply:
We don't, you can still pass any reference, see the unit test. Only if it was `'static` it would always pass ownership (or require that it's a static reference).